packages/common/src/core/schemas/internal/getLocationOptions.ts

```diff
@@ -29,87 +22,45 @@ export async function getLocationOptions(
   portalName: string,
   hubRequestOptions: IHubRequestOptions
 ): Promise<IHubLocationOption[]> {
-  const typeFromEntity = getTypeFromEntity(entity);
-
-  switch (typeFromEntity) {
-    // case "content":
-    //   return getContentLocationOptions(entity);
-    default:
-      return getDefaultLocationOptions(entity, portalName, hubRequestOptions);
-  }
-}
-
-// TODO: Refactor parameters, they are icky gross
-// TODO: Maybe move these to outside of core and into respective entities
+  const defaultExtent: IExtent = await getGeographicOrgExtent(
```

Comment:
I know you didn't chose this name, but it would be best if it was named `orgExtent`